dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -11,13 +11,65 @@
     = t(:slow_loading)
     %br
     %a{href: 'javascript: location.reload();'}= t(:try_reloading)
-  - if @level.game == Game.netsim
+
+-# Now define the dependencies for the level we are loading.
+-# These will be written at the end of the body tag so that we render as much
+-#   of the page as possible before running scripts, but so that browsers can
+-#   intelligently optimize downloading resources.
+- computed_app_options = app_options
+- apps_path = computed_app_options[:baseUrl]
+- app_name = computed_app_options[:app]
+- is_droplet = computed_app_options[:droplet]
+- is_netsim = @level.game == Game.netsim
+- is_blockly = !is_droplet && !is_netsim
```

Comment:
That convention [applies to predicate methods](https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark).  Here, `is_droplet`, `is_netsim` and `is_blockly` are local variables. [You can't use a question mark in the name of a variable](http://stackoverflow.com/questions/1263160/ruby-boolean-attribute-naming-convention-and-use).  If these get refactored into methods, I'll definitely change the names.

Fun fact: One reason I haven't done that refactor yet is that `app_options` has a terrifying side effect that breaks everything if you call it more than once in a single request.  Try it out!